java/src/com/mongodb/csfle/CSFLEHelpers.java

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.mongodb.csfle;
```

Comment:
Consider moving this into a sub-package.  Reason being: in IntelliJ you get tons of warnings that the methods and constants can be package-private.  It's very distracting.  We should be warning-free in IntelliJ.  Moving this class to a different package will get rid of those warnings.  Alternatively, make everything package-private by removing the `public` modifier everywhere.

Reply:
Ok. I am moving this to a subpackage. I will install IntelliJ after this upcoming commit to follow up on any warnings that I might spot.